src/localStorage.js

```diff
@@ -0,0 +1,70 @@
+const taskContainer = document.querySelector('[data-todos]');
+const localTaskKey = 'task.list';
+const localTasks = JSON.parse(localStorage.getItem('task.list')) || [];
+
+const clearElement = (element) => {
+  while (element.firstChild) {
+    element.removeChild(element.firstChild);
+  }
+};
+
+const save = () => {
+  localStorage.setItem(localTaskKey, JSON.stringify(localTasks));
+};
+
+const removeTask = (e) => {
+  const trash = e.target;
+  if (trash.tagName.toLowerCase() === 'span') {
+    // eslint-disable-next-line eqeqeq
+    const selectedTrash = localTasks.filter((task) => task.id !== trash.id);
+    localTasks.splice(trash.id, 1);
+    localStorage.setItem('task.list', JSON.stringify(selectedTrash));
+    window.location.reload();
+  }
+};
+
+const editTask = (e) => {
+  const edit = e.target;
+  if (edit.tagName.toLowerCase() === 'textarea') {
+    // eslint-disable-next-line eqeqeq
+    const selectedEdit = localTasks.find((task) => task.id == edit.id);
+    selectedEdit.description = edit.value;
+  }
+  save();
```

Comment:
- [ ] I would recommend adding a new file for the edit and remove functionality. As we have to Use ES6 modules to write modular JavaScript.